.gitignore

```diff
@@ -6,3 +6,4 @@ npm-debug.log
 *.swp
 .idea
 coverage
+*.map
```

Comment:
If you had this to the ignore we will be removing the functionality added with this PR #148